common/components/ui/Dropdown.tsx

```diff
@@ -17,15 +18,15 @@ export default class Dropdown extends React.Component<Props> {
     selectedOption: this.props.value || '',
     hasBlurred: false
   };
-
-  public componentDidUpdate(_: Props, prevState: State) {
-    const { value } = this.props;
-    const { selectedOption } = prevState;
-
-    if (selectedOption !== value) {
-      this.setState({ selectedOption: value });
-    }
-  }
+  //
```

Comment:
It would be preferable to remove commented out code.

Reply:
done